src/utils/cutChips.js

```diff
@@ -0,0 +1,18 @@
+export const cutChips = (chips, maxLength, onclick) => {
+  if (chips.length > maxLength) {
+    let hiddenChipsNumber = '+' + (chips.length - maxLength)
```

Comment:
Let's try and use template literals:
```suggestion
    let hiddenChipsNumber = `+${chips.length - maxLength}`
```